rust-toolchain

```diff
@@ -1 +1 @@
-1.41.1
+1.41.0
```

Comment:
Nope.

Reply:
lol, I wondered last night when I was going to sleep if I'd left this in :)